specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -459,7 +459,41 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{accountName}/skus": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CognitiveServices/skus": {
+      "get": {
+        "tags": [
+          "Skus",
+          "AvailabilitySets"
```

Comment:
not sure `AvailabilitySets` make sense here, if not please remove it

Reply:
Right, that does not make sense, put CognitiveServicesAccounts there.